nss-cohort-36/bangazon-client-virtual-magicians-1
# Jg product 2

# Description
This adds a GET method on the API manager.

Closes #[Enter issue #]
## Type of change

Please delete options that are not relevant.
- [ x] New feature (non-breaking change which adds functionality)

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jg-product-2`
4. code on the client should show the GET method created for a single object

# Checklist:
